File: Spring_POS/Spring_BackEnd/src/main/java/lk/ijse/spring/dto/OrderDetailsDTO.java
package lk.ijse.spring.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@AllArgsConstructor
@NoArgsConstructor
@Data
@ToString
public class OrderDetailsDTO {
    private String oid;
    private String itemCode;
    private int qty;
    private double unitPrice;
    private double total;
}

File: Spring_POS/Spring_FrontEnd/model/OrderDetailsDTO.js
function OrderDetailsDTO(orderID, code, quantity, uPrice, tot){
    let oid = orderID;
    let itemCode = code;
    let qty = quantity;
    let unitPrice = uPrice;
    let total = tot;

    this.getORDOrderId = function (){
        return oid;
    }
    this.getORDCode = function (){
        return itemCode;
    }
    this.getORDQty = function (){
        return qty;
    }
    this.getORDUPrice = function (){
        return unitPrice;
    }
    this.getORDTot = function (){
        return total;
    }

    this.setORDOrderID = function (orderID){
        oid = orderID;
    }
    this.setORDItmCode = function (code){
        itemCode = code;
    }
    this.setORDQty = function (quantity){
        qty = quantity;
    }
    this.setORDUPrice = function (uPrice){
        unitPrice = uPrice;
    }
    this.setORDTot = function (tot){
        total = tot;
    }
}